javascript/builtins/intl/DurationFormat.json

```
@@ -0,0 +1,229 @@
+{
+  "javascript": {
+    "builtins": {
+      "Intl": {
+        "DurationFormat": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat",
+            "spec_url": "https://tc39.es/proposal-intl-duration-format/#durationformat-objects",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "deno": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
```

Need to update to 16.4. throughout. Also, "mirror" should be used for Safari (and other browsers)

```suggestion
              "safari": {
                "version_added": "16.4"
              },
              "safari_ios": "mirror",
```